.github/workflows/test-php.yml

```diff
@@ -0,0 +1,67 @@
+name: Tests PHP
+
+on:
+  push:
+    paths-ignore: ["docs/**"]
```

Comment:
All of these should probably limit to `on.push.branches = [master]` and `on.pull_request.branches = [master]`